src/server/server.ts

```diff
@@ -598,7 +598,9 @@ namespace ts.server {
             if (option && value) {
                 switch (option) {
                     case "-file":
-                        logEnv.file = stripQuotes(value);
+                        const { logPath, extraPartCounter } = getLogPath(i + 1);

```

Comment:
should not we do this one level higher and do it on `value` in the main `for` loop instead? makes it easier for future additions.

Reply:
Moved it up one level and renamed.